tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,614 @@
+        token                  = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+        cluster_ca_certificate = base64decode(scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate)
+        load_config_file       = false
+    }
+
+    provider "helm" {
+        kubernetes {
+            host                   = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].host
+            token                  = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+            cluster_ca_certificate = base64decode(scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate)
+        }
+    }
+    ```
+
+2. To configure the distribution of the pods on the cluster we will use labels set on nodes. To get the label list of a node, you can use the `kubectl` command below. You can see that multiple labels, sets by the `scaleway-cloud-controller-manager`, contain information about the `region` and `zone` where the node is deployed. The label `topology.kubernetes.io/zone`, at the end of the list, is particularly interesting in your case as it will allow us to know on which availability zone a pod will be deployed.
+    ```
+    kubectl get node scw-kapsule-multi-az-pool-fr-par-1-51c432554c4 \
+        --template '{{range $k, $v := .metadata.labels}}{{printf "%s=%s\n" $k $v}}{{end}}'
+
+    beta.kubernetes.io/arch=amd64
+    beta.kubernetes.io/instance-type=PRO2-XXS
+    beta.kubernetes.io/os=linux
+    failure-domain.beta.kubernetes.io/region=fr-par
+    failure-domain.beta.kubernetes.io/zone=fr-par-1
+    k8s.scaleway.com/kapsule=6ee0a129-ee3b-4a89-ad4b-a74754441746
+    k8s.scaleway.com/managed=true
+    k8s.scaleway.com/multi-az=
+    k8s.scaleway.com/node=51c43255-4c42-449a-9e63-d734c357cd65
+    k8s.scaleway.com/pool=2d669c67-bb46-4e4f-93a0-a59834f6c6ba
+    k8s.scaleway.com/pool-name=pool-fr-par-1
+    k8s.scaleway.com/runtime=containerd
+    kubernetes.io/arch=amd64
+    kubernetes.io/hostname=scw-kapsule-multi-az-pool-fr-par-1-51c432554c4
+    kubernetes.io/os=linux
+    node.kubernetes.io/instance-type=PRO2-XXS
+    topology.csi.scaleway.com/zone=fr-par-1
+    topology.kubernetes.io/region=fr-par
+    topology.kubernetes.io/zone=fr-par-1
+    ```
+
+3. You want to have at least two `ingress-nginx` pods per Availability Zone. To do so, set `replicaCount` to `6`. However, this alone won't guarantee distribution across zones. To that end, use `topologySpreadConstraints` to instruct the Kubernetes scheduler to distribute replicas based on the `topology.kubernetes.io/zone` label, which can have values like `fr-par-1`, `fr-par-2`, or `fr-par-3`. You can find detailed documentation for [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) in the Kubernetes documentation. Also there's an anti affinity to prevent the `ingress-nginx` pod to be scheduled on the same node in each AZ.
```

Comment:
```suggestion
3. Ensure you have at least two `ingress-nginx` pods per Availability Zone. To do so, set `replicaCount` to `6`. However, this alone won't guarantee distribution across zones. To that end, use `topologySpreadConstraints` to instruct the Kubernetes scheduler to distribute replicas based on the `topology.kubernetes.io/zone` label, which can have values like `fr-par-1`, `fr-par-2`, or `fr-par-3`. You can find detailed documentation for [Pod Topology Spread Constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/) in the Kubernetes documentation. Also there's an anti affinity to prevent the `ingress-nginx` pod to be scheduled on the same node in each AZ.
```